tests/components/cast/test_media_player.py

```diff
@@ -755,14 +725,14 @@ async def test_entry_setup_list_config(hass: HomeAssistantType):
         await cast.async_setup_entry(hass, MockConfigEntry(), None)
 
     assert len(mock_setup.mock_calls) == 2
-    assert mock_setup.mock_calls[0][1][1] == {"host": "bla"}
-    assert mock_setup.mock_calls[1][1][1] == {"host": "blu"}
+    assert mock_setup.mock_calls[0][1][1] == {"uuid": "bla"}
+    assert mock_setup.mock_calls[1][1][1] == {"uuid": "blu"}
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6299130)